contents/handbook/people/hiring-process/manager-hiring.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Manager Hiring
+sidebar: Handbook
+showTitle: true
+hideAnchor: true
+---
+
+## Manager hiring at PostHog
+
+* The best source of leads is within the company
+  * We know they understand the details.
+  * We know they're a cultural fit.
+  * This helps retain good people already working here that may want to try this out.
+  * It's easier to teach a great individual contributor the basic management skills than it is the other way around.
+  * More of a two way door - we can move people back if they already work here, but we probably would have to let someone go if they joined specifically to manage.
+* They have to be good at the individual contributor work in that team - the manager can't be the person that fills a huge gap in the team. This is more inspiring.
```

Comment:
I'm not 100% sure what this sentence means - they need to be a good IC but also if they leave shouldn't fill a huge gap?